### ISSUE TITLE
Gradle cannot find this plugin

### ISSUE THREAD

**Issue by gerob311** [glureau/dokka-mermaid#3]
I've included `dokkaPlugin("com.glureau:html-mermaid-dokka-plugin:0.4.5")` in my dependencies. When I try to run the dokkaHTML build, I get the following error:

```
Execution failed for task ':dokkaHtml'.
> Could not resolve all files for configuration ':dokkaHtmlPlugin'.
   > Could not find com.glureau:html-mermaid-dokka-plugin:0.4.5.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/com/glureau/html-mermaid-dokka-plugin/0.4.5/html-mermaid-dokka-plugin-0.4.5.pom
     Required by:
         project :

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
```

I've included the mavenCentral repository. What is the correct repository?

**Comment by glureau** [comment]
Thanks for your feedback, looks like the version is not on maven central, it should, so not sure if I failed the deployment or if it has been removed. Investigating. In the meantime you can pick the 0.4.4 that is available on mavencentral. I'll publish soon new updates.

**Comment by glureau** [comment]
I've re-published 0.4.5 and also release 0.4.6 (using mermaid 10.2 the latest version available). I can't verify that those 2 versions are working at the moment and I've serious doubts, so please if you decide to use those versions leave me a comment if it's ok for you 🙏🏻 .
